src/Nox.Generator/Infrastructure/Persistence/DtoDbContext.template.cs

```diff
@@ -52,9 +54,11 @@ internal class DtoDbContext : DbContext
         _interceptors = interceptors;
     }
 
+    {{- if codeGeneratorState.Solution.Domain?.Entities != null }}
```

Comment:
this is not needed the template will not be called, and Entities are an empty collection

if this is not working as expected, lets see it together

Reply:
Fixed